src/test/javascript/testBookScript.js

```diff
@@ -0,0 +1,73 @@
+describe('Testing bookScript.js - processURL', () => {
+  let apiKey;
+  beforeEach(function () {
+    apiKey = 'xyz';
+  })
+
+  it('test if it parses the correct URL when author and title are set', () => {
+    let title = 'Title';
+    let author = 'Author';
+    let isbn = null;
+    let url = 'https://www.googleapis.com/books/v1/volumes?q=intitle:Title+inauthor:Author&country=UK&key=xyz';
+    expect(processURL(title, author, isbn, apiKey)).toEqual(url);
```

Comment:
nit: Here and below: These local varialbles feel a bit redundant. 

expect(processURL("mytitle", "myauthor", /*isbn=*/null, apiKey)).toEqual("https://");

Note how I used a parameter comment to clarify what the third argument is (as null doesn't say much).
https://google.github.io/styleguide/jsguide.html#formatting-param-name-comments